lib/plurimath/asciimath_parser/cos.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Plurimath
+  module AsciimathParser
```

Comment:
Maybe just `Asciimath` instead of `AsciimathParser`?

I'm thinking you may have this module hierarchy:

```ruby
Plurimath::Math # for math representation (this is the "central model" of math)
Plurimath::Math::Formula # one math formula
Plurimath::Asciimath # the AsciiMath parser
Plurimath::Latexmath # the LaTeX math parser
Plurimath::Omml # the OMML parser
Plurimath::Mathml # the MathML parser
```

And then you have the objects per math language:
```ruby
Plurimath::Asciimath::Function::Cos
Plurimath::Asciimath::Symbol
...
```

Reply:
@ronaldtse @suleman-uzair I was thinking that we might not need the functions for mathematical representation. So, we can move them to the common Math namespace. For example, we can have:

```ruby
Plurimath::Math::Function::Cos
```

because these functions are related to central model and we don’t need them for every representation.
Let me know if I am missing something here?

Reply:
@HassanAkbar indeed. I meant `Plurimath::Asciimath::Function::Cos` as a parse node of the Asciimath language "node" of representing `Plurimath::Math::Function::Cos`. If we don't need it, fine.